app/package.json

```diff
@@ -12,15 +12,15 @@
     "start": "node start.js"
   },
   "dependencies": {
-    "express": "^4.18.2",
-    "express-validator": "^6.14.2",
     "body-parser": "^1.18.3",
     "cookie-parser": "^1.4.6",
+    "express": "^4.18.2",
+    "express-validator": "^6.14.2",
     "front-matter": "^4.0.2",
     "glob": "^8.0.3",
-    "govuk-elements-sass": "3.1.3",
     "govuk_frontend_toolkit": "^9.0.1",
     "govuk_template_jinja": "^0.26.0",
+    "govuk-elements-sass": "3.1.3",
```

Comment:
Nice little tidy up.

I would've expected `govuk-elements-sass` to stay where it was since hyphens come before underscores in ASCII/UTF (think it also just makes sense from a human-readable point of view - "e" comes before "f", and all that).

Reply:
@domoscargin I'll put my hand up and say I manually ordered it wrong last time (sorry), but this time it was definitely **npm** fixing my mistake so I've trusted it here 😬

There's an ESLint plugin for keeping `package.json` files in order, maybe we need it 🙈 

Thanks @domoscargin